CONTRIBUTING.md

```diff
@@ -324,14 +328,51 @@ Secretless supports using [Go plugins](https://golang.org/pkg/plugin/) to extend
 its functionality. To learn about writing new Secretless plugins and for more
 information on the types of plugins we currently support, visit the [plugin API directory](pkg/secretless/plugin).
 
-### Submodules
+## Submodules
```

Comment:
Do other sections at this level get listed in the TOC? If so, this would need to be added there.